app/decorators/publication_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class PublicationDecorator < Draper::Decorator
+  delegate_all
+
+  def title_link
```

Comment:
You can add file link here too. Take a look at DataUploadDecorator, on Publications show page there should be downloadable link for the file.